catalog-rest-service/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -461,14 +461,67 @@
       },
       "additionalProperties": false
     },
+    "columnProfilerConfig": {
```

Comment:
Shouldn't this be in `createTable.json` as well?

Reply:
added another endpoint for tableProfilerConfig ```/{id}/tableProfilerConfig``` this will be a PUT endpoint and delete endpoint. 
IMO createTable.json should contain the main content about a table, and everything else, such as tableProfile or tableProfilerConfigs are extensions so will be part of other apis